sort-dependencies-gradle-plugin/src/main/kotlin/com/squareup/sort/SortDependenciesTask.kt

```diff
@@ -40,6 +40,11 @@ abstract class SortDependenciesTask @Inject constructor(
   @get:Input
   abstract val mode: Property<String>
 
+  @get:Optional
+  @get:Option(option = "strict", description = "Enables strict mode.")
```

Comment:
🔕  Trivial suggestion on documentation - 
```suggestion
  @get:Option(option = "strict", description = "Fails the task if the sort order cannot be verified/updated. A parse error could cause this. Default is true.")
```

Reply:
I feel like "strict mode" is universally understood in the case of such tools, and adding a non exhaustive list of examples is not necessary, but I'm ok to change it if @autonomousapps confirms.
Also, the current default value is `false`. Should we change it? (I'm for keeping it opt-in)

Reply:
Default should be false. Description should be
```kotlin
"Enables strict mode, which will cause this task to fail if there is an error parsing or sorting the dependencies. Default is false."
```